server/belga/io/feed_parsers/base_belga_newsml_1_2.py

```diff
@@ -85,8 +85,6 @@ def parse(self, xml, provider=None):
                     # Slugline and keywords is epmty
                     item['slugline'] = None
                     item['keywords'] = []
-                    # delete subject is duplicated
-                    item['subject'] = [dict(t) for t in {tuple(d.items()) for d in item['subject']}]
```

Comment:
why it's deleted?

Reply:
We use set here so it will not work for nested dict such as `country`
e.g `{'name': 'France', 'qcode': 'country_fra', 'scheme': 'country', 'translations': {'name': {'nl': 'Frankrijk', 'fr': 'France'}}}`
but it seems [itertools.groupby](https://docs.python.org/3/library/itertools.html#itertools.groupby) do the work so we have updated it